code/controllers/communications.dm

```diff
@@ -126,20 +126,20 @@ var/const/MED_I_FREQ = 1485
 var/const/SEC_I_FREQ = 1475
 
 var/list/radiochannels = list(
-	"Common"		= PUB_FREQ,
+	"Public"		= PUB_FREQ,
 	"Science"		= SCI_FREQ,
 	"Command"		= COMM_FREQ,
 	"Medical"		= MED_FREQ,
 	"Engineering"	= ENG_FREQ,
 	"Security" 		= SEC_FREQ,
-	"Response Team" = ERT_FREQ,
-	"Special Ops" 	= DTH_FREQ,
-	"Mercenary" 	= SYND_FREQ,
+	"Epsilon-11" 	= ERT_FREQ,
+	"Nu-7" 			= DTH_FREQ,
```

Comment:
We use Hammer-Down for deathsquad.

Reply:
Nu7 is Hammer Down. : )

Reply:
This pull makes that explicit actually, it was just "death commando" before.

![image](https://user-images.githubusercontent.com/37093159/47136588-fabe3000-d268-11e8-9888-ff46e69fae4c.png)